demo/qwik-app/src/routes/index.tsx

```diff
@@ -1,25 +1,35 @@
-import type { DocumentHead } from '@qwik.dev/router';
-import { component$, useStyles$ } from '@qwik.dev/core';
-import styles from './index.css?inline';
+import type { DocumentHead } from "@qwik.dev/router";
+import { component$, useStyles$ } from "@qwik.dev/core";
+import styles from "./index.css?inline";
 
 export default component$(() => {
-	useStyles$(styles);
+  useStyles$(styles);
 
-	return (
-		<div>
-			<h1>Google provider</h1>
-			<div>Poppins</div>
-			<p>Press Start 2P</p>
-		</div>
-	);
+  return (
+    <div>
+      <h1>Google provider</h1>
```

Comment:
```suggestion
      <h1>Test with different providers</h1>
```

Reply:
I don't understand this suggestion ... 
I listed some providers like a Google, Bunny Fontshare and Fontsource...